tensorflow/core/kernels/generate_box_proposals_op.cu.cc

```diff
@@ -38,56 +38,66 @@ limitations under the License.
 
 namespace tensorflow {
 typedef Eigen::GpuDevice GPUDevice;
-#define TF_OP_REQUIRES_CUDA_SUCCESS_ASYNC(context, result, done)       \
+#define TF_OP_REQUIRES_CUDA_SUCCESS(context, result)                   \
   do {                                                                 \
     cudaError_t error(result);                                         \
     if (!SE_PREDICT_TRUE(error == cudaSuccess)) {                      \
       context->SetStatus(errors::Internal("Cuda call failed with",     \
                                           cudaGetErrorString(error))); \
-      done();                                                          \
       return;                                                          \
     }                                                                  \
   } while (0)
 
 namespace {
+
+template <bool T>
+__device__ float legacy_op(float);
```

Comment:
Use camel case for function names, also use more descriptive name e.g. ApplyOffset.

Reply:
Done